core-it-support/maven-it-helper/src/main/java/org/apache/maven/it/AbstractMavenIntegrationTestCase.java

```diff
@@ -568,33 +568,19 @@ protected Verifier newVerifier( String basedir, String settings, boolean debug )
 
             String path = settingsFile.getAbsolutePath();
 
-            // dedicated CLI option only available since MNG-3914
-            if ( matchesVersionRange( "[2.1.0,)" ) )
+            verifier.addCliOption( "--global-settings" );
+            if ( path.indexOf( ' ' ) < 0 )
             {
-                verifier.addCliOption( "--global-settings" );
-                if ( path.indexOf( ' ' ) < 0 )
-                {
-                    verifier.addCliOption( path );
-                }
-                else
-                {
-                    verifier.addCliOption( '"' + path + '"' );
-                }
+                verifier.addCliOption( path );
             }
             else
             {
-                verifier.getSystemProperties().put( "org.apache.maven.global-settings", path );
+                verifier.addCliOption( '"' + path + '"' );
             }
```

Comment:
Do we still need to quote ourselves? Feels weird, no?

Reply:
I wouldn't like touch it until Verifier 2.0

Reply:
Yes, note that the upgrade to verifier 2.0 will break quotes arguments.  Also note that I plan to merge https://github.com/apache/maven-integration-testing/pull/176 soon and it includes an upgrade to verifier 2.0-SNAPSHOT.  Unless we really do want to postpone until it's released, but the snapshots are available on public apache repos, so that's not a dealbreaker, especially as ITs are never released.

Reply:
We have many changes in Verifier 2.0.0, many deprecated methods, etc 
My proposition is to release Verifier 2.0.0-M1 as is now - I can do it in next weekend.

In next step after #176   we can clean of using deprecated methods we can release next milestone with next part of resolved issues and so on, until to final 2.0.0

As always I like step by step and if it possible split bigger changes.

What do you think?

Reply:
Sounds like a plan to me !